pkg/exchange/okex/parse.go

```diff
@@ -24,7 +24,13 @@ func parseWebSocketEvent(in []byte) (interface{}, error) {
 	}
 	switch {
 	case e.IsOp():
-		return e.WebSocketOpEvent, nil
+		// need unmarshal again because arg in both WebSocketOpEvent and WebSocketPushDataEvent
```

Comment:
Why you need unmarshal again? 
You have said that: `// "op" and "PushData" are exclusive.`

Reply:
I believe that `err := json.Unmarshal(in, &e)` is enough

Reply:
I tried. But It really need unmarshal again, 
<img width="1025" alt="Screenshot 2023-10-03 at 14 30 32" src="https://github.com/c9s/bbgo/assets/10070491/89e75740-90f0-46df-acf0-cab9b492be64">

Reply:
Looks like you want to delay json encoding. According to the doc, you should apply json.RawMessage https://pkg.go.dev/encoding/json#RawMessage

Reply:
No, I just want to Unmarshal one time, to distinguish whether it is `WebSocketPushDataEvent` or `WebSocketOpEvent` , but I don't know why, actually it can know it is `WebSocketPushDataEvent` or `WebSocketOpEvent` , but every field in `Arg` can't be recognized. I guess that's because `Arg` both exist in `WebSocketPushDataEvent` and `WebSocketOpEvent` struct field ?
Unmarshal twice can solve this problem.